staging/src/k8s.io/kubectl/pkg/metricsutil/metrics_printer.go

```diff
@@ -65,14 +65,15 @@ func (n *NodeMetricsSorter) Len() int {
 
 func (n *NodeMetricsSorter) Swap(i, j int) {
 	n.metrics[i], n.metrics[j] = n.metrics[j], n.metrics[i]
+	n.usages[i], n.usages[j] = n.usages[j], n.usages[i]
 }
 
 func (n *NodeMetricsSorter) Less(i, j int) bool {
 	switch n.sortBy {
 	case "cpu":
 		qi := n.usages[i][v1.ResourceCPU]
 		qj := n.usages[j][v1.ResourceCPU]
-		return qi.Value() > qj.Value()
+		return qi.MilliValue() > qj.MilliValue()
```

Comment:
Why we need to go with `MilliValue` here? Are you aware this can overflow thus causing more errors?

Reply:
Because CPU metrics are always collected with `MilliValue` (instead of MEMORY metrics which are collected with `Value`)

For exemple, for one sample node, `MilliValue` will return _1628_ and `Value` will return _2_. (Value = MilliValue / 1000 round up)

Another exemple show you existing code at the end of _metrics_printer.go_ file : 

```
func printSingleResourceUsage(out io.Writer, resourceType v1.ResourceName, quantity resource.Quantity) {
	switch resourceType {
	case v1.ResourceCPU:
		fmt.Fprintf(out, "%vm", quantity.MilliValue())
	case v1.ResourceMemory:
		fmt.Fprintf(out, "%vMi", quantity.Value()/(1024*1024))
	default:
		fmt.Fprintf(out, "%v", quantity.Value())
	}
}

```

Reply:
Fair, thanks for the details.